package.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "templated-hashmap",
+  "version": "2.0.1",
```

I believe this should match the git tag name.
```suggestion
  "version": "v2.0.1",
```